src/averageAges.js

```diff
@@ -37,7 +51,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people.filter(woman => woman.sex === 'f');
+  let avgAge = [];
+
+  if (withChildren !== undefined) {
```

Comment:
```suggestion
  if (withChildren) {
```
Порівняння до undefined можна забрати